go.mod

```diff
@@ -3,9 +3,16 @@ module github.com/Kable-io/kable-go
 go 1.19
 
 require (
-	github.com/golang/protobuf v1.4.2 // indirect
-	golang.org/x/net v0.0.0-20200822124328-c89045814202 // indirect
-	golang.org/x/oauth2 v0.0.0-20210514164344-f6687ab2804c // indirect
-	google.golang.org/appengine v1.6.6 // indirect
-	google.golang.org/protobuf v1.25.0 // indirect
+	github.com/aws/aws-sdk-go v1.40.16 // indirect
```

Comment:
to the extent possible, let's try to remove as many of these as possible: 
* <3 skinny libraries
* avoid version conflicts
* less to maintain over time